examples/resources/opslevel_service/resource.tf

```
@@ -15,7 +15,7 @@ data "opslevel_tier" "tier3" {
 resource "opslevel_team" "foo" {
   name             = "foo"
   responsibilities = "Responsible for foo frontend and backend"
-  aliases          = ["bar", "baz"]
+  aliases          = ["foo", "bar", "baz"] # NOTE: if set, value of "name" must be included
```

not entirely true more like

```suggestion
  aliases          = ["foo", "bar", "baz"] # NOTE: if set, slugified value of "name" must be included
```

---

Theres a few more too